src/stores/Annotation/Annotation.js

```diff
@@ -252,6 +252,7 @@ export const Annotation = types
     draftSelected: false,
     autosaveDelay: 5000,
     isDraftSaving: false,
+    isSuggestionsAccepting: false,
```

Comment:
could you please add a comment describing this flag? from the name alone it's not very clear what it supposed to do